include/crocoddyl/core/actions/unicycle.hxx

```diff
@@ -97,4 +97,16 @@ void ActionModelUnicycleTpl<Scalar>::set_cost_weights(const typename MathBase::V
   cost_weights_ = weights;
 }
 
+template <typename Scalar>
+const Scalar& ActionModelUnicycleTpl<Scalar>::get_dt() const {
+  return dt_;
+}
+
+template <typename Scalar>
+void ActionModelUnicycleTpl<Scalar>::set_dt(const Scalar& dt) {
```

Comment:
Do we want const-ref on primitive types? Same for the return in `get_dt`

Reply:
@wxmerkt - thanks for spot it.
We need to also adapt the Python bindings

Reply:
You want primitive types to be copied ?

Reply:
Yes - for primitive types it's often as expensive to create the reference and pass and resolve it than to copy its value. There was some discussion on this here in the past as well and all const-ref for primitives was changed to pass-by-value

Reply:
Ok, thanks for the explanation. I'll update the code.

Reply:
Do you have the link to the discussion ?

Reply:
- Mostly here in comments by myself and Justin: https://github.com/loco-3d/crocoddyl/pull/642
- More uses of it were removed in https://github.com/loco-3d/crocoddyl/pull/893